h/features/client.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.auth import role
+from h.features import models
+
+
+class UnknownFeatureError(Exception):
+    pass
+
+
+class Client(object):
+    """
+    Determine if the named feature is enabled for the current request.
+    If the feature has no override in the database, it will default to
+    False. Features must be documented, and an UnknownFeatureError will be
+    thrown if an undocumented feature is interrogated.
+    """
+
+    def __init__(self, request, fetcher=models.Feature.all):
+        self.request = request
+        self._fetcher = fetcher
+        self._cache = None
+
+    def __call__(self, name):
+        return self.enabled(name)
+
+    def enabled(self, name):
+        """
+        Determine if the named feature is enabled for the current request.
+
+        If the feature has no override in the database, it will default to
+        False. Features must be documented, and an UnknownFeatureError will be
+        thrown if an undocumented feature is interrogated.
+
+        When the internal cache is empty, it will automatically load the
+        feature flags from the database first.
+        """
```

Comment:
Is this docstring just a duplicate of the class one? On a class like this, it might be okay to just docstring the class and not the method. Then `enabled()` could maybe just be moved into `__call__()` as well